app/components/topic-card/template.hbs

```diff
@@ -4,11 +4,34 @@
   {{/if}}
   {{topic.description}}
 </div>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
-  <div class='icon upvote {{if isUpvoter 'active'}}'></div>
-  <div class='count'>{{topic.upvoteCount}}</div>
-</button>
-<button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'volunteer'}}>
-  <div class='icon volunteer {{if isVolunteer 'active'}}'></div>
-  <div class='count'>{{topic.volunteerCount}}</div>
-</button>
\ No newline at end of file
+
+{{#if topic.talkBy}}
+  <div class="volunteer-container">
+    <img class='avatar' alt="{{topic.talkBy.username}}" title="{{topic.talkBy.username}}" src='{{topic.talkBy.avatar}}' />
+  </div>
+   {{date-format topic.topicDate format="MMMM Do"}}
+   {{#if talkByCurrentUser}}
+     <button class="link-btn" {{action "cancelTalkBy"}}>
+      [{{fa-icon 'times'}}]
+     </button>
+    {{/if}}
+{{else}}
+  <button class='toggle-button-counter' disabled={{session.notAuthenticated}} {{action 'upvote'}}>
+    <div class='icon upvote {{if isUpvoter 'active'}}'></div>
+    <div class='count'>{{topic.upvoteCount}}</div>
+  </button>
+  <button class="icon-btn" alt="Claim topic" title="Claim topic" {{action "chooseTalkDate"}} disabled={{session.notAuthenticated}}>
```

Comment:
Just noticed, I think this is the last thing... since `{{action "chooseTalkDate"}}` has no param, if today is not a meetup day it never passes a value into `chooseTalkDate` and so the date is left empty and it doesn't actually get scheduled.

I think fixing this bug is going to require some minor changes to this logic within `chooseTalkDate` as well.